src/App.js

```diff
@@ -1,15 +1,31 @@
-import React from 'react'
-
-import FetchFromApi_event from './components/events'
-import FetchFromApi_member from './components/members'
-
-const App = () => {
+import React, { useState, useEffect } from "react";
+import MemberPage from "./pages/memberPage";
+import EventPage from "./pages/eventPage";
+import "./styles/App.css";
+function App() {
+  const [pageRoute, setPageRoute] = useState(true);
+  useEffect(() => {
+    if (!pageRoute) {
+      document.querySelector("#eve").classList.add("activeLink");
+      document.querySelector("#mem").classList.remove("activeLink");
+    } else {
+      document.querySelector("#mem").classList.add("activeLink");
+      document.querySelector("#eve").classList.remove("activeLink");
+    }
+  }, [pageRoute]);
   return (
-    <div>
-<FetchFromApi_event/>
-<FetchFromApi_member/>
+    <div className="App">
+      <div className="controlBoard">
+        <p id="eve" onClick={() => setPageRoute(!pageRoute)}>
```

Comment:
Use conditional classes here....avoid manipulation of Real DOM

Reply:
In the updated version conditional classes are used.